src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -399,11 +360,42 @@ function createFiberFromElementType(
       type == null ? type : typeof type,
       info,
     );
+    fiber.pendingProps = element.props;
+  }
+
+  fiber.pendingWorkPriority = priorityLevel;
+
+  if (__DEV__) {
+    fiber._debugSource = element._source;
+    fiber._debugOwner = element._owner;
   }
+
   return fiber;
-}
+};
+
+exports.createFiberFromFragment = function(
+  elements: ReactFragment,
+  internalContextTag: TypeOfInternalContext,
+  priorityLevel: PriorityLevel,
+): Fiber {
+  // TODO: Consider supporting keyed fragments. Technically, we accidentally
+  // support that in the existing React.
```

Comment:
Do we? We might have in Stack, but that's no longer the "existing React." :D If we don't support it in 16, let's not worry about it.

Reply:
This was an existing TODO comment in the code :D. If it's no longer relevant, should I remove it?